libvast/include/vast/defaults.hpp

```diff
@@ -245,6 +245,9 @@ constexpr std::chrono::milliseconds signal_monitoring_interval
 constexpr std::chrono::milliseconds initial_request_timeout
   = std::chrono::seconds{10};
 
+/// Timeout for the scheduler to give up on a partition.
+constexpr std::chrono::milliseconds scheduler_timeout = std::chrono::hours{1};
```

Comment:
Isn't that extremely long for a single partition lookup? How about one minute?

Reply:
I'm not sure what the correct value is, I went high because it was "infinite" before and I didn't want to change it too much. 1 minute feels too short, if we're running on a machine with a slow filesystem and high load I think it can be more than that. Maybe 15 minutes is a good starting point.

Reply:
Also good, let's make it 51.

Reply:
We need to make sure this does not cause shutdown to no longer function similar to what we had in https://github.com/tenzir/vast/pull/2191#discussion_r845153665